lib/themes/dosomething/paraneue_dosomething/js/components/ContextSlide.js

```diff
@@ -1,5 +1,9 @@
 const React = require('react');
 
+/**
+ * ContextSlide Component
+ * <ContextSlide />
+ */
 class ContextSlide extends React.Component {
   constructor(props) {
```

Comment:
It was in there from before... Might leave for now if that's cool, since I didn't delve into building farther on that yet. Not sure if I will have to or not.